components/CaseStatus/EditCaseStatusForm/ReviewEditCaseStatusForm.spec.tsx

```diff
@@ -48,6 +98,54 @@ describe('ReviewEditCaseStatusForm', () => {
     ).toBeInTheDocument();
   });
 
+  it('displays the form CP - end', () => {
+    const { getByText } = render(
+      <ReviewEditCaseStatusForm
+        title="Review case status details"
+        personId={mockedResident.id}
+        caseStatusType="CP"
+        caseStatusId={123}
+        action="end"
+        formAnswers={{
+          endDate: '2020-12-01',
+        }}
+      />
+    );
+
+    expect(getByText('Review case status details')).toBeInTheDocument();
+    expect(getByText('01 Dec 2020')).toBeInTheDocument();
+    expect(
+      getByText('Do you want to end this case status?')
+    ).toBeInTheDocument();
+  });
+
+  it('displays the form LAC - end', () => {
+    const { getByText } = render(
+      <ReviewEditCaseStatusForm
+        title="Review case status details"
+        personId={mockedResident.id}
+        caseStatusType="LAC"
+        caseStatusId={123}
+        action="end"
+        formAnswers={{
+          endDate: '2020-12-01',
+          episodeReason: 'E4A',
+        }}
+      />
+    );
+
+    expect(getByText('Review case status details')).toBeInTheDocument();
+    expect(getByText('01 Dec 2020')).toBeInTheDocument();
+    expect(
+      getByText(
+        'E4A: Returned home to live with parent(s), relative(s), or other person(s) with parental responsibility as part of the care planning process (not under a special guardianship order or residence order or (from 22 April 2014) a child arrangement order).'
+      )
+    ).toBeInTheDocument();
+    expect(
+      getByText('Do you want to end this case status?')
+    ).toBeInTheDocument();
+  });
+
   it('displays start date and notes for the edit', () => {
```

Comment:
I'm unsure of the value of this tests now that you've updated the other test for Edit in CIN? (Line 8)

Could we add the check for the startDate into that test instead?

Reply:
It mainly checks the general visualisation of the component!

Checking the startDate is another way of doing that, either way they're CIN-related properties only :)